brandon_tan/index_scraper.py

```diff
@@ -0,0 +1,85 @@
+import pandas as pd
```

Comment:
Is this module 100% your own code? I feel like I have seen this before. If so, please add a reference to the original source.

Reply:
Yes this is 100% my own code that I have used in a previous internship.